bc_obps/registration/models.py

```diff
@@ -188,13 +188,15 @@ class Meta:
 class Address(models.Model):
     """Address model"""
 
-    street_address = models.CharField(max_length=1000, db_comment="Street address of relevant location)")
-    municipality = models.CharField(max_length=1000, db_comment="Municipality of relevant location")
+    street_address = models.CharField(max_length=1000, null=True, db_comment="Street address of relevant location)")
+    municipality = models.CharField(max_length=1000, null=True, db_comment="Municipality of relevant location")
     province = CAProvinceField(
-        db_comment="Province of the relevant location, restricted to two-letter province postal abbreviations"
+        db_comment="Province of the relevant location, restricted to two-letter province postal abbreviations",
+        null=True
     )
     postal_code = CAPostalCodeField(
-        db_comment="Postal code of relevant location, limited to valid Canadian postal codes"
+        db_comment="Postal code of relevant location, limited to valid Canadian postal codes",
+        null=True
```

Comment:
We are using Address as a foreign key in other tables, so I think we can make those fields nullable instead of making all fields in the Address data model nullable.